src/server/scheduling_tbd/interviewing/interviewers.clj

```diff
@@ -329,6 +337,18 @@
          (get-an-answer (merge ctx interviewer-q)))
        (finally (ws/send-to-chat {:dispatch-key :interviewer-busy? :value false :client-id client-id}))))
 
+;TODO: add for other interviews
```

Comment:
My vision here is that every interview will implement an _analyze-response_ function that looks at every response individually to do something to it (or not).